CHANGELOG.md

```diff
@@ -52,6 +58,19 @@ BUG FIXES:
  * scheduler: Fixed a bug where jobs requesting multiple CSI volumes could be incorrectly scheduled if only one of the volumes passed feasibility checking. [[GH-10143](https://github.com/hashicorp/nomad/issues/10143)]
  * ui: Fixed the rendering of interstitial components shown after processing a dynamic application sizing recommendation. [[GH-10094](https://github.com/hashicorp/nomad/pull/10094)]
 
+## 1.0.5 (Unreleased)
```

Comment:
Since 1.0.5 is unreleased yet and will be released prior to (or with) 1.1.0, I opted to keep these to-be-backported PRs only in 1.0.5.

Reply:
I could've sworn we had this 1.0.5 section already... we'll need to do the rest of https://github.com/hashicorp/nomad/pulls?q=is%3Apr+label%3Astage%2Fneeds-backporting+is%3Aclosed but I'll hit those.